docs/design/fabric-v2+/fpc-registration.puml

```diff
@@ -114,18 +129,24 @@ group enclave creation
 
   end group
 
-  return Attestation
+  Enclave1 -> Enclave1 : Sealed_State <- seal <CC_Params, Host_Params, Private_Keys, Public_Keys>
```

Comment:
Maybe also make a group around this like for enclave attestation as this will be a module with the store call under the cover?  (insofar, good question whether we even want to show the `ocall` (e.g., we do not show ocalls for attestation either?)
PS: the same of course should then also apply to fpc-cc-invocation for symmetry reason ...

Apropos consistency and groups around sub-modules:
- the call to AttestationToEvidence(Credeitnails.Attestation) should probably be _inside_ the conversion box, not before?
- further down, the 'check Crednetials.Evidence probably should be called `verify_evidence` to be consistent with api/interfaces.attsation.md

Reply:
Updated the suggestions regarding attestation.

Regarding sealing and storage let's postpone the changes here as we have not yet defined the corresponding module yet! I added a comment

Reply:
i guess we could have made the box/group even without having the API specified in details but i'm definitely not religous about it ...